frontend/src/employee-mobile-frontend/components/attendances/child-info/ChildInfo.tsx

```diff
@@ -282,3 +281,15 @@ function getColorByStatus(status: AttendanceStatus) {
     ? colors.accents.water
     : colors.blues.medium
 }
+
+function getBackgroundColorByStatus(status: AttendanceStatus) {
+  return status === 'ABSENT'
+    ? '#E8E8E8'
+    : status === 'DEPARTED'
+    ? '#B3C5DD'
+    : status === 'PRESENT'
+    ? '#EEF4B3'
+    : status === 'COMING'
+    ? '#E2ECF2'
+    : colors.blues.medium
+}
```

Comment:
These colors don't seem to exist in the design system. Should they? 🤔

Reply:
Don't think so. They are only defined for this exact usecase in figma. If they are later used somewhere else we need to move them to our theme files.